packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -712,6 +712,7 @@ class TimeInput extends React.Component {
           [
             <Input
               {...inputAttributes}
+              aria-label={this.context.intl.formatMessage({ id: 'Meridiem Display' })}
```

Comment:
Given that this input is read only, I'm thinking we need aria-readonly on this input element. http://www.maxability.co.in/2015/11/aria-readonly-property/